### PR TITLE
Creating new educational program

### PR DESCRIPTION
**Pull request by simassimonas**
Added a new create-educational-program component

### CODE REVIEW

File: src/app/components/educational-programs/create-educational-program/create-educational-program.component.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  addPosition() {
+    const position = this.fb.group({
+      positionName: ['', [Validators.required]],
+      descriptionAndRequirements: ['', [Validators.required]],
+    });
+
+    this.positions.push(position);
+  }
+
+  deletePosition(i: any) {
```
Comment (AudriusBaltusis) [inline src/app/components/educational-programs/create-educational-program/create-educational-program.component.ts:53]: type is `number`

File: src/app/components/educational-programs/create-educational-program/create-educational-program.component.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    this.positions.removeAt(i);
+    console.log('detePosition, i = ', i);
```
Comment (AudriusBaltusis) [inline src/app/components/educational-programs/create-educational-program/create-educational-program.component.ts:55]: please do not use `console.log` in production code. If it is necessary for some reason, add comment about it

File: src/app/components/educational-programs/educational-programs.component.ts [hunk 3]
```diff
@@ -1,69 +1,103 @@
...
+
+export interface EducationalProgram {
+  name: string;
+  acceptanceFrom: string;
+  acceptanceTo: string;
+  programFrom: string;
+  programTo: string;
+}
+
+const ELEMENT_DATA: EducationalProgram[] = [
```
Comment (AudriusBaltusis) [inline src/app/components/educational-programs/educational-programs.component.ts:11]: Not critical, but I suggest move mock data and interfaces away from component to separate files

**Review by AudriusBaltusis — APPROVED** [review]

**Comment by AudriusBaltusis** [comment]
Please resolve conflicts

File: src/app/components/educational-programs/create-educational-program/create-educational-program.component.ts (hunk 2 above)
Comment (simassimonas) [reply src/app/components/educational-programs/create-educational-program/create-educational-program.component.ts:55]: My bad, forgot to delete it after testing

**Pull request merged**